edgelet/aziot-edged/src/app.rs

```diff
@@ -14,51 +14,24 @@ use crate::error::{Error, ErrorKind, InitializeErrorReason};
 use crate::logging;
 
 #[allow(deprecated)]
```

Comment:
nit: what's this tag for? any chance we can remove it as part of this PR?

Reply:
No idea what it was for. Build succeeds without it. Removed.